index.js

```diff
@@ -115,9 +115,11 @@ module.exports = {
 			}
 		],
 		'no-alert': 'error',
+		"no-autofix/prefer-const": "error",
```

Comment:
Not requested by original issue, but may be useful. @nomego what do you think?

Reply:
Oh, I misunderstood how to use this, it should not be there. :blush:

Reply:
Removed this.

Reply:
I haven't gotten annoyed by the autofix'es at all yet, and I normally write consts anyway :)
But if anyone's annoyed by it we can try and see if we're more or less annoyed by eslint reporting errors and not fixing them :)